### DISCUSSION TITLE
Hardware compatibility

### DISCUSSION THREAD

**Discussion by RobinBeismann** [jomjol/AI-on-the-edge-device#1666] [Q&A] [answered]
Hey,

I don't have usable wifi in my basement but could easily run an Ethernet line from my network cabinet down there.
I stumbled upon this, as I understand it has the required Hardware in IT, right?

https://eckstein-shop.de/M5Stack-PoE-Camera-W5500-EN

I saw it's EOL but that shouldn't matter.

Thanks,
Robin

**Comment by jomjol** [comment]
Would have the necessary components, but there is no implementation for this hardware.
We do not plan to support several hardwares, due to limited resources. But if you can transfer, we are happy to integrate.

**Reply by RobinBeismann** [reply]
Thanks for the fast response! I somewhat thought the hardware would be the same as the M5Stack has a "ESP32-D0WDQ6-V3" but now I see that the hardware compatibility list explicitly lists the "ESP32-D0WDQ6 (revision 1)" as core. So I assume that one isn't compatible?

I'm new to the whole ESP32 world so please forgive my stupid questions. :)

**Comment by jomjol** [comment, marked as answer]
Well, I'm not totally sure. But as we do not have any LAN drivers implemented, I assume, that it needs to be modified and most propalby also dedicated compiled.

**Reply by RobinBeismann** [reply]
I went for the bundle of three ESP32-CAMs that's linked on the hardware compatibility wiki instead. I'll just place an old 2.4 GHz Unifi AP in the basement which I replaced years ago against newer models.
Thanks again!
